src/Storageless/Http/SessionMiddleware.php

```diff
@@ -216,19 +220,23 @@ private function appendToken(SessionInterface $sessionContainer, Response $respo
         return $response;
     }
 
-    private function shouldTokenBeRefreshed(?Token $token) : bool
+    private function shouldTokenBeRefreshed(?Token $token): bool
     {
         if (! ($token && $token->hasClaim(self::ISSUED_AT_CLAIM))) {
             return false;
         }
 
-        return $this->timestamp() >= $token->getClaim(self::ISSUED_AT_CLAIM) + $this->refreshTime;
+        $issuedAt = $token->getClaim(self::ISSUED_AT_CLAIM);
+
+        assert(is_numeric($issuedAt));
```

Comment:
I suggest to add `ini-values: zend.assertions=1` to `shivammathur/setup-php` on GA workflow, otherwise this would be a mere psalm notation with no useful feedbacks

Reply:
Not a.major issue, since it's verified by mutation tests.

It is indeed meant as a psalm assertion